flipperdevices/flipperzero-firmware
# [FL-3600] Added `fal_embedded` parameter for PLUGIN apps

# What's new

- fbt, ufbt: refactored external apps internal state management
- fbt, ufbt: added `fal_embedded` parameter for PLUGIN apps, to embed them into .fap

# Verification 

- set `fal_embedded=True` for a plugin in example_advanced_plugins
- install app
- observe the plugin in `apps_assets`

# Checklist (For Reviewer)
